esx_service/cli/vmdkops_admin.py

```diff
@@ -839,6 +839,9 @@ def generate_tenant_ls_rows(tenant_list):
         else:
             default_datastore = vmdk_utils.get_datastore_name(tenant.default_datastore_url)
 
+        if default_datastore is None:
+            return rows
```

Comment:
so it seem default_datastore could be "" (line 838) and None.  Why use 2 representations for the same ?